CHANGELOG.rst

```diff
@@ -46,7 +46,7 @@ This document describes changes between each past release.
 - Changed psycopg2 dependency for psycopg2-binary. (#1905)
 - Renamed core notions (ie. record and collection) (#710)
 - JSON Schema validation is optimized by keeping instances of validator cached. (#1807)
-
+- Remove delete-collection command (#1959)
```

Comment:
nitpick: I wouldn't list this change in the *internal changes* category. Probably *breaking changes*...